rule.go

```diff
@@ -914,3 +911,14 @@ func (r *Rule) HasVar(s string) bool {
 	}
 	return false
 }
+
+//getSidMgs returns a string representing a sidmsg.map entry
+func (r *Rule) getSidMgs(s string) string {	
+	var sidmsg = fmt.Sprintf("%s || %s", string(r.SID), r.Description);
+	for _, ref := range r.References {
+		if ref.Type == "url" {
```

Comment:
This overwrites the beginning part of the string that you want.
Please consider using a strings.Builder[0] or a `[]string` and using a `.join()` in the final return.

[0] https://golang.org/pkg/strings/#Builder.String

Reply:
Yes, my bad, should be solved now.